test/models/user_test.rb

```diff
@@ -0,0 +1,9 @@
+require "test_helper"
+
+describe User do
+  let(:user) { User.new }
+
+  it "must be valid" do
+    value(user).must_be :valid?
+  end
```

Comment:
Not a valid test and it fails! A new User with the validations you've defined on it ( `  validates :username, presence: true` ) would not be valid.

Also, I'd love to know what the method `value` does! I haven't seen that before.